research/a3c_blogpost/a3c_cartpole.py

```diff
@@ -0,0 +1,390 @@
+import os
+os.environ["CUDA_DEVICE_ORDER"] = "PCI_BUS_ID"   # see issue #152
+os.environ["CUDA_VISIBLE_DEVICES"] = ""
+
+
+import threading
+import gym
+import multiprocessing
+import numpy as np
+from queue import Queue
+import argparse
+import matplotlib.pyplot as plt
+
+
+import tensorflow as tf
+from tensorflow.python import keras
+from tensorflow.python.keras import layers
+
+tf.enable_eager_execution()
+print("Eager execution: {}".format(tf.executing_eagerly()))
+
+def str2bool(v):
```

Comment:
I think this is nice on production code but not so nice on code that we expect people to read carefully. Just use type=bool

Reply:
One option is:

```
parser.add_argument("--no_train", action="store_false", dest="train", default=True)
```

The only advantage is that it's a tiny bit less code.